src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        if (userService.findByEmail(email) != null) {
+            return userService.findByEmail(email).getPassword().equals(password);
+        }
         return false;
```

Comment:
you duplicate `userService.findByEmail(email)` let save it to variable and write it using one return

Reply:
@MentorMateAcademy, fixed, thanks for pointing on that!